pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -4879,6 +4879,7 @@ func TestReconciler_ErrorFinalizingPod(t *testing.T) {
 }
 
 func TestIsPodOwnerManagedByQueue(t *testing.T) {
+	defer jobframework.EnableIntegrationsForTest(t, "batch/job", "ray.io/raycluster")()
```

Comment:
```suggestion
	t.CleanUp(jobframework.EnableIntegrationsForTest(t, "batch/job", "ray.io/raycluster"))
```

Reply:
done

Reply:
Please do the same in all the usages of this function.

Reply:
It's just a best practice. We should get into the habit of using the appropriate tools. In this case it doesn't make a difference, but in other cases it could. Just being consistent can direct people writing tests in the future to follow the same pattern.